web/package.json

```diff
@@ -15,6 +15,7 @@
     "@redwoodjs/forms": "7.0.2",
     "@redwoodjs/router": "7.0.2",
     "@redwoodjs/web": "7.0.2",
+    "humanize-string": "2.1.0",
```

Comment:
If you separate the commits where you add dependencies, it can be easier to know what was added and why later.